Lib/pdb.py

```diff
@@ -162,13 +162,14 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, skip=None,
         # Read $HOME/.pdbrc and ./.pdbrc
         self.rcLines = []
         if readrc:
-            if 'HOME' in os.environ:
-                envHome = os.environ['HOME']
-                try:
-                    with open(os.path.join(envHome, ".pdbrc")) as rcFile:
-                        self.rcLines.extend(rcFile)
-                except OSError:
-                    pass
+            try:
+                # bpo-20523: $HOME does not exist on Windows 7,
+                # use os.path.expanduser()
```

Comment:
This is just me being picky about comments, so use (or not) the following at your will:

1 - Although this has been vastly done in CPython, I'm personally against including the issue number (bpo-20523) at the code changes. This practice pollutes the decode and thinking forward, I wouldn't like to see a million `bpo's` in the source code ten years from now.

2 - The second line `use os.path.expanduser()` is redundant. A good comment should tell `why` that change is necessary, and not `how` you are accomplishing that. Your code should be readable enough for anyone to figure out the `hows`. Which in this case, it's already readable enough.

Reply:
@augustogoulart 
1. we have 431 "bpo-XYZ: " in the code of CPython.
2. I explain the reason of `os.path.expanduser`, because $HOME does not exist on Windows 7.